src/tools/rustfmt/ci/integration.sh

```diff
@@ -53,67 +54,72 @@ function check_fmt_base {
         return 1
     fi
     cat rustfmt_output
-    ! cat rustfmt_output | grep -q "internal error"
-    if [[ $? != 0 ]]; then
+    grep -q "internal error" < rustfmt_output
```

Comment:
The lint was complaining about the `cat` redirection which could be replaced by `< rustfmt_output`. And then I removed the `!` operator and reverted the checked value (from `!= 0` to `== 0`). Less tricky to understand (if the command didn't fail, ie `grep` found the string, then not good). ^^'